Mathlib/Analysis/SpecialFunctions/Integrals.lean

```diff
@@ -91,7 +91,8 @@ theorem intervalIntegrable_rpow' {r : ℝ} (h : -1 < r) :
     rw [intervalIntegrable_iff] at m ⊢
     refine' m.congr_fun _ measurableSet_Ioc; intro x hx
     rw [uIoc_of_le (by linarith : 0 ≤ -c)] at hx
-    simp only [Pi.smul_apply, Algebra.id.smul_eq_mul, log_neg_eq_log, mul_comm,
+    rw [Pi.smul_apply]
```

Comment:
Let's leave a comment on all of these `rw [Pi.smul_apply]`, so it's possible to clean them all up at once if we ever sort out the problem.